source/shared/net/concepts/concepts-baker.rst

```diff
@@ -36,15 +38,40 @@ A baker needs to stake a part of its CCD balance on the baker account. Later, th
    If an account owns an amount that was transferred with a release schedule,
    the amount can be staked even if it hasn't been released yet.
 
-To be chosen for baking a block, the baker must take part in a
-*lottery*. The greater the stake the greater is the baker's chance of winning the lottery and being selected to bake a block.
+To be chosen to bake a block, the baker must take part in a
+*lottery*. The greater the baker's stake, the greater the baker's chance of winning the lottery and being selected to bake a block.
 
-The same stake is used when calculating whether a baker is included in the  :ref:`finalization <glossary-finalization>` committee or not.
+The same stake is used when calculating whether a baker is included in the :ref:`finalization <glossary-finalization>` committee or not.
 
 Time concepts
-----------------
+-------------
 The Concordium blockchain divides time into :ref:`epochs <glossary-epoch>` and :ref:`slots <glossary-slot>`.
 
-When considering the rewards and other baking-related concepts, the concept of an *epoch* is used as a unit of time that defines a period in which the set of current bakers and stakes are fixed. Epochs have a duration of 1 hour and the duration is fixed at the Genesis block.
+.. image:: ../images/concepts/epochs-slots.png
+   :alt: timeline showing how 14400 slots make up one epoch
+
+When considering the rewards and other baking-related concepts, the concept of an *epoch* is used as a unit of time that defines a period in which the set of current bakers and stakes are fixed. Epochs have a duration of 1 hour and the duration is fixed at the :ref:`Genesis block <glossary-genesis-block>`.
+
+Epochs are subdivided into slots. On any given :ref:`branch <glossary-branch>`, each slot can have a maximum of one block, but multiple blocks on different branches can be produced in the same slot. Slots have a duration of 250ms, and the duration is fixed at the :ref:`Genesis block <glossary-genesis-block>`.
+
+A :ref:`cool-down period <glossary-cool-down-period>` describes a period of time during which certain activities or transactions are frozen. For example, if you decrease a baker stake in the desktop wallet, the stake will be decreased after a cool-down period. The cool-down period is 2 epochs. During the cool-down period, you’ll not be able update the stake. After the cool-down period, the amount by which you decreased your stake is returned to your disposable balance.
```

Comment:
Isn't the cool-down period currently `166 epochs ~ 1 week` on Mainnet?

Reply:
According to Thomas it varies based on the transaction.